script.js

```diff
@@ -1,12 +1,138 @@
-//You can edit ALL of the code here
+let container = document.getElementById("root");
+let searchContainer = document.getElementById("search-container");
+let allEpisodes;
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  makeDropdownMenu(allEpisodes);
+  searchForEpisode();
```

Comment:
Since you've made `allEpisodes` a global variable, you don't need to pass it in as a parameter on lines 6 and 7. Those functions can access any variables in global scope themselves.